### PR TITLE
General PDE solving with NNs fix revert

### PR DESCRIPTION
**Pull request by KirillZubov**
fix revert

### CODE REVIEW

File: src/NeuralNetDiffEq.jl [hunk 1]
```diff
@@ -4,7 +4,9 @@ using Reexport, Statistics
 @reexport using DiffEqBase
 
 using Flux, Zygote, DiffEqSensitivity, ForwardDiff, Random, Distributions
-using DiffEqFlux, Adapt, CuArrays, StochasticDiffEq
+using DiffEqFlux, Adapt, CuArrays
```
Comment (ChrisRackauckas) [inline src/NeuralNetDiffEq.jl:7]:
```suggestion
using DiffEqFlux, Adapt, CuArrays, StochasticDiffEq
```

**Comment by ChrisRackauckas** [comment]
Looks like CI is having challenges. Put a print in with the usings to make it not time out.

**Comment by KirillZubov** [comment]
> Looks like CI is having challenges. Put a print in with the usings to make it not time out.

@ChrisRackauckas  could you please describe more details about what to do

**Comment by ChrisRackauckas** [comment]
I'll want to talk to you about the structure of the setup, but that can come later, let's get v1 in and then start tweaking it.

**Comment by KirillZubov** [comment]
> I'll want to talk to you about the structure of the setup, but that can come later, let's get v1 in and then start tweaking it.

ok, so do we just merge this PR or what to do with the challenge with CI?

**Comment by ChrisRackauckas** [comment]
Test failure is stochastic and unrelated, but merging. But @KirillZubov could you try and make that test a bit more robust?

**Comment by KirillZubov** [comment]
> > I'll want to talk to you about the structure of the setup, but that can come later, let's get v1 in and then start tweaking it.
> 
> ok, so do we just merge this PR or what to do with the challenge with CI?

yeah, sure, in next PR

**Pull request merged**